### PR TITLE
Improve error message for input grid without any active cells.

### PR DESCRIPTION
**Pull request by blattms**
We now give a different (more explanatory) error if the input grid has no active cells at all. Often this points to a user error and having a more precise error message might help here.

### CODE REVIEW

**Comment by blattms** [comment]
jenkins build this please

File: opm/simulators/utils/readDeck.cpp [hunk 1]
```diff
@@ -326,27 +326,40 @@ namespace {
...
-                return true;
+                return {true,
+                    R"(No active cell in input grid has valid/finite cell geometry
+Please check geometry keywords and modifications to e.g. pore volume,
+especially if the grid is imported through GDFILE.)"};
             }
         }
 
-        return false;
+        return {false, ""};
```
[the remark this answers is not in the document]
Comment (blattms) [reply opm/simulators/utils/readDeck.cpp]: fixed.

**Comment by blattms** [comment]
jenkins build this please

**Review by bska — APPROVED** [review]
Thanks a lot for the updates and for taking steps to emit better diagnostics.  I'll merge this into master.

**Pull request merged**